nodes/scheduling/descheduler/nodes-descheduler-release-notes.adoc

```diff
@@ -10,7 +10,36 @@ The {descheduler-operator} allows you to evict pods so that they can be reschedu
 
 These release notes track the development of the {descheduler-operator}.
 
-:operator-name: The {descheduler-operator}
-include::snippets/operator-not-available.adoc[]
-
 For more information, see xref:../../../nodes/scheduling/descheduler/index.adoc#nodes-descheduler-about_nodes-descheduler-about[About the descheduler].
+
+[id="descheduler-operator-release-notes-5.1.0"]
+== Release notes for {descheduler-operator} 5.1.0
+
+Issued: 30 October 2024
+
+The following advisory is available for the {descheduler-operator} 5.1.0:
+
+* link:https://access.redhat.com/errata/RHEA-2024:6341[RHEA-2024:6341]
+
+[id="descheduler-operator-5.1.0-new-features"]
+=== New features and enhancements
+
+* Two new descheduler profiles are now available:
++
+--
+** `CompactAndScale`: This profile attempts to evict pods from underutilized nodes to allow a workload to run on a smaller set of nodes.
+** `LongLifecycle`: This profile balances resource usage between nodes and enables the `RemovePodsHavingTooManyRestarts` and `LowNodeUtilization` strategies.
+--
```

Comment:
@ingvagabund one question, do we need to mention about evictionLimit as well in the doc ?

Reply:
`evictionLimit` was introduced to help CNV team to reduce the number of eviction in progress. It's useful to mention the new [parameter](https://github.com/openshift/cluster-kube-descheduler-operator?tab=readme-ov-file#parameters). Yet, not required.

Reply:
okay, got it !! Thanks !! Reason i have asked is because we have tested and have added test coverage for it, so.

Reply:
@ingvagabund @kasturinarra So is this something we want to mention in the main descheduler docs? Or something specific only for CNV?

Reply:
Main descheduler docs. There's no harm if others take advantage of this.